tests/test_api_users.py

```diff
@@ -202,9 +198,6 @@ async def test_username_select(server, reg_token_client):
 ]
 invalid_usernames_put = [
     'foò',  # unicode
-    'fo=o',  # invalid char
-    'fo o',  # space
-    'f\'oo',  # quote
```

Comment:
These usernames can no longer be created in newer keycloak versions, which breaks test_user_put_invalid()

Reply:
Does it break on `client = await rest(username)`?  I guess then it's useful to split this into two lists, the second of which goes to `test_invalid_usernames()` and `test_username_invalid()`.